obsei/preprocessor/text_cleaner.py

```diff
@@ -0,0 +1,108 @@
+import logging
+from typing import Any, Dict, List, Optional
+import string
+import re
+from unicodedata import normalize as unicode_decode
+
+import nltk
+from nltk.corpus import stopwords
+from nltk.stem import PorterStemmer
+from nltk.tokenize import word_tokenize
+
+from pydantic import PrivateAttr
+
+from obsei.analyzer.base_analyzer import AnalyzerRequest
+from obsei.preprocessor.base_text_cleaner import (
+    BaseTextCleaner,
+    BaseTextCleanerConfig,
+)
+
+logger = logging.getLogger(__name__)
+nltk.download('punkt')
+
+
+class TextCleaner(BaseTextCleaner):
+
+    def __init__(self, **data: Any):
```

Comment:
As mentioned here all things should be supplied via constructor params so user can control them. And also these params should have sane defaults.
https://github.com/lalitpagaria/obsei/issues/75#issuecomment-849307243

Reply:
@shahrukhx01 it would be great if we can give choice to user (ofcourse we should have sane defaults as well)